src/index.html

```diff
@@ -0,0 +1,181 @@
+                <li class="lang">
+                    <p>
+                        Languages
+                    </p>
+                    <img class="imglang" src="../assets/arrowdown.png" alt="arrow">
+                </li>
+                <div class="tech-list">
+                    <li class="lang-skill">
+                                    <img src="../assets/js.png" alt="js">
+                                    <p>
+                                        JavaScript
+                                    </p>
+                                </li>
+                    <li class="lang-skill">
+                                    <img src="../assets/css.png" alt="html">
+                                    <p>
+                                        CSS
+                                    </p>
+                                </li>
+                    <li class="lang-skill">
+                                    <img src="../assets/html.png" alt="css">
+                                    <p>
+                                        HTML
+                                    </p>
+                                </li>
+                </div>
+                <li class="lang">
+                    <p>
+                        FrameWorks
+                    </p>
+                    <img class="imglang" src="../assets/arrowside.png" alt="">
+                </li>
+                <li class="line"></li>
+                <li class="lang">
+                    <p>
+                        Skills
+                    </p>
+                    <img class="imglang" src="../assets/arrowside.png" alt="">
+                </li>
+                <li class="line"></li>
+            </ul>
+        </div>
+    </section>
+    <section id="contact" class="contact-me">
+        <div class="bg-image">
+            <h2 class="contact-title">Contact me</h2>
+            <p class="contact-text">
+                If you have an application you are interested in developing, a feature
+                that you need built or a project that needs coding. I’d love to help
+                with it
+            </p>
+            <form action="https://formspree.io/f/xqkgeweb" method="POST" class="contact-form">
+                <input type="text" name="name" id="username" placeholder="Your Name" maxlength="30" required />
+                <input type="email" name="email" id="useremail" placeholder="Your Email" required />
+                <textarea name="message" id="usermessage" cols="30" rows="10" maxlength="500"
+                    placeholder="Write your message here" required></textarea>
+                    <p class="alert"></p>
+                <div class="button-resume btn-form">
+                    <button class="btn" type="submit">Get in touch</button>
+                </div>
+            </form>
+        </div>
+    </section>
+
+    <div class="modal-example">
+        
+        
+      </div>
+
+    <script src="main.js"></script>
+</body>
+
```

Comment:
- Hey @ gdvl777:wink: You are doing really good but you need to pay closer attention to the suggestions the previous reviewer offered you as it appears there aren't many changes in this submission.
- [ ] Kindly see this [video](https://watch.screencastify.com/v/9n1LjB4on7xHM3YtJWrK) I recorded with my experience on your portfolio website. As indicated by the first reviewer, it has quite a lot of responsiveness issues on both big and small screens which are not good for the user experience.
- [ ] I suggest you take a look at the video 3 times, compare with the Figma design, and then see how to implement the change using relative units for your widths, using `display:flex` for your positioning to ease things up

Reply:
> * Hey @ gdvl777😉 You are doing really good but you need to pay closer attention to the suggestions the previous reviewer offered you as it appears there aren't many changes in this submission.
> * [ ]  Kindly see this [video](https://watch.screencastify.com/v/9n1LjB4on7xHM3YtJWrK) I recorded with my experience on your portfolio website. As indicated by the first reviewer, it has quite a lot of responsiveness issues on both big and small screens which are not good for the user experience.
> * [ ]  I suggest you take a look at the video 3 times, compare with the Figma design, and then see how to implement the change using relative units for your widths, using `display:flex` for your positioning to ease things up

I can not watch your video.